sources/packages/api/src/route-controllers/application/models/application.model.ts

```diff
@@ -264,11 +259,9 @@ export const transformToApplicationDetailDto = (
     applicationInstitutionName: applicationDetail.location?.name,
     applicationPIRStatus: applicationDetail.pirStatus,
     applicationAssessmentStatus: applicationDetail.assessmentStatus,
-    applicationCOEStatus: applicationDetail.coeStatus,
     applicationFormName: applicationDetail.programYear.formName,
     applicationProgramYearID: applicationDetail.programYear.id,
     applicationPIRDeniedReason: getPIRDeniedReason(applicationDetail),
-    applicationCOEDeniedReason: getCOEDeniedReason(applicationDetail),
```

Comment:
i guess we are using applicationCOEStatus and applicationCOEDeniedReason in the progress bar of the student application in trackStudentApplication.json . Are we removing them too?

Reply:
good catch, yes we need to update the application details API too

Reply:
COEDenied reason and COEStatus added here from the firstCOE of the application.